eradiate/scenes/biosphere/_discrete.py

```diff
@@ -465,7 +465,7 @@ def l_vertical(self):
 
 @parse_docs
 @attr.s
-class ConicalLeafCloudParams(LeafCloudParams):
+class ConeLeafCloudParams(LeafCloudParams):
```

Comment:
I thought I had changed this in my branch :thinking: Now I'm confused :D

Reply:
Apparently it creeped into `main` :p If this doesn't break anything for you, I'll proceed with merging.

Reply:
Ahhh I found this and added this modification in another branch. It's fine you can merge :)